specification/translation/Azure.AI.TextTranslation/tspconfig.yaml

```
@@ -13,10 +13,10 @@ emit:
   - "@azure-tools/typespec-autorest"
 options:
   "@azure-tools/typespec-autorest":
-    azure-resource-provider-folder: "../data-plane/Azure.AI.TextTranslation"
+    azure-resource-provider-folder: "data-plane"
     emitter-output-dir: "{project-root}/.."
     examples-directory: "./examples"
-    output-file: openapi.json
+    output-file: "{azure-resource-provider-folder}/Azure.AI.TextTranslation/stable/v3.0/openapi.json"
```

```suggestion
    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json"
```

We should be able to use this config, if we also change the namespace from `TextTranslation` to `Azure.AI.TextTranslation`.

---

@mikeharder I think that change is very likely to break SDKs generated from this.  Also, we cannot recreate the 'v3.0' from the interpolated variable, the versions tring will be '3.0'  personally, I think the version string is fine, but putting the literal string for the service in t config seems like a lesser evil than changing the namespace

---

OK, let's keep these custom `output-file` strings in the services that need them for now, and we can revisit this later in a push to make the specs as consistent as possible.